projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -71,7 +71,7 @@ export class CollectionService<S extends EntityState<any, string>>  {
   }
 
   /** The path to the collection in Firestore */
-  get path(): string | Observable<string> {
```

Comment:
@GrandSchtroumpf ok so it turns out that removing this line causes type errors also Subcollection service. So maybe it's worth removing SubcollectionService at this point, anyway it's not compatible to current CollectionService code.

Reply:
And I can write an util function, like we discussed before, using same logic found in SubcollectionService

Reply:
I would like not to introduce too much breaking changes for next version. Please consider submitting a PR that doesn't breaks `Subcollection`.

Reply:
Sure that's possible, but as we already discussed, the SubcollectionService is not working correctly with current v2, and leaving it as deprecated will mean it's working, just not recommended  to use, which is confusing, cause it's not. So either you need to rewrite it for backwards compatibility, which probably isn't worth it, or just remove by introducing a breaking change, but we also can document how to easily migrate old code, by introducing util functions.

Reply:
Ok, let's do that. Can you update the Breaking change file at the root of the project to explain the migration ?
Thx